packages/codership/mysql-galera/overlay/app-readme.md

```diff
@@ -0,0 +1,13 @@
+# MySQL Galera from Codership
+
+This is a Helm chart for MySQL Galera from Codership.
+
+## Installation
+    - Plan your cluster capacity. Default is 3 nodes, each with 1 vCPU and 4GB of RAM, the storage is 2GB. It is minimal requirement for Galera to work.
+    - Specify a new namespace for the MySQL Galera cluster, like `mysql-galera`.
+    - Install the MySQL Galera chart.
```

Comment:
These bullet points are indented too much. The result is that this appears as a code block:
![image](https://github.com/user-attachments/assets/d8075233-d190-4df8-9976-d1d4526285cf)

Reply:
fixed the wrong indentation